doc/source/release-notes/Percona-Server-8.0.15-6.rst

```diff
@@ -71,11 +73,11 @@ Bugs Fixed
 - :table:`INFORMATION_SCHEMA.GLOBAL_TEMPORARY_TABLES` queries could crash if
   online ``ALTER TABLE`` was running in parallel. Bug fixed :psbug:`5566`.
 
-- setting :variable:`log_slow_verbosity` and enabling the
-  :variable:`slow_query_log` could lead to a server crash. Bug fixed
+- setting :variable:`log_slow_verbosity` to a non-default value and enabling
+  the :variable:`slow_query_log` could lead to a server crash. Bug fixed
   :psbug:`4933`.
 
-- :ref:`compression_dictionary` operations were not handled under
+- :ref:`compression_dictionary` operations would result in an error under
```

Comment:
They still result in an error, as they should be (the operations write and innodb-force-recovery make server read-only). The fix changed non-intuitive error message to a better one

Reply:
compression_dictionary operations were not handled properly under innodb-force-recovery. Now they work correctly under innodb_force_recovery <= 2, and are forbidden with innodb_force_recovery >= 3 (needs editing for style)

Reply:
comment not addressed